.github/workflows/publish-docker-image.yml

```diff
@@ -17,10 +17,18 @@ jobs:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
 
+      - name: Get current Studio version
+        id: get-studio-version
+        uses: martinbeentjes/npm-get-version-action@v1.3.1
```

Comment:
Could you change the version to a commit SHA based version?
Something like `docker/build-push-action@f2a1d5e99d037542a71f64918e516c093c6f3fc4`

Reply:
@Shurtu-gal Sure, done!